pilot/pkg/proxy/envoy/filters/filters.go

```diff
@@ -31,61 +31,65 @@ import (
 	alpn "istio.io/istio/security/proto/envoy/config/filter/http/alpn/v2alpha1"
 )
 
-const OriginalSrc = "envoy.listener.original_src"
+const (
+	OriginalSrcFilterName = "envoy.listener.original_src"
+	// Alpn HTTP filter name which will override the ALPN for upstream TLS connection.
+	AlpnFilterName = "istio.alpn"
+)
 
 // Define static filters to be reused across the codebase. This avoids duplicate marshaling/unmarshaling
 // This should not be used for filters that will be mutated
 var (
-	corsFilter = &hcm.HttpFilter{
+	CorsFilter = &hcm.HttpFilter{
```

Comment:
nit: Now that these have been moved to `filters` package we can remove `Filter` from the name to avoid name stutter